qarakusi.am.py

```diff
@@ -118,8 +118,8 @@ def load_scene(module, scene_class_name, prefix=False):
         scene_names = [key for key in dir(module)
                        if key.startswith(scene_class_name)
                        and isinstance(getattr(module, key), type)
-                       and getattr(module, key) is not Scene
-                       and issubclass(getattr(module, key), Scene)]
+                       and issubclass(getattr(module, key), Scene)
+                       and getattr(module, key).__module__ == module.__name__]
```

Comment:
I'm checking if the scene is defined in the module we are looking in (e.g. `Scene` from `manim` won't satisfy, so I removed the unnecessary extra condition)